src/toast-notification/classes.scss

```diff
@@ -67,6 +67,35 @@
       animation: closeAnimation 120ms cubic-bezier(0.4, 0, 1, 1) both;
     }
   }
+
+  &.iui-placement-top-start {
+    right: unset;
```

Comment:
Why do you need unsets?

Reply:
Removed unsets. Moved all placement stuff to appropriate classes.